src/SearchService.php

```diff
@@ -42,7 +43,12 @@ public function search(string $keyword = "", int $start = 0, int $limit = 10): ?
 
                 // If the file is anything but HTML, then it's downloadable. Ensure we append the file type and file size to the end
                 if ($fileType != self::FILE_TYPE_HTML) {
-                    $title = $this->formatFileTitle($title, $fileType, $result['fileSize']);
+                    $file = $this->getFileFromURL($result['indexUrl']);
+                    $title = $this->formatFileTitle(
+                        !$file ? 'File Not Found' : $file->Title,
```

Comment:
Can you flip this around please so the default is to use `$file->Title` rather than using `!$file`? Thanks!